libs/xmlrpc-c/Makefile

```diff
@@ -8,12 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xmlrpc-c
-PKG_VERSION:=1.51.07
+PKG_VERSION:=1.60.02
```

Comment:
Why do we need to have advanced version based on https://xmlrpc-c.sourceforge.io/downloading.php ?

Reply:
We don't. I just wanted to keep the branch consistent with [Buildroot](https://github.com/buildroot/buildroot/blob/master/package/libxmlrpc/libxmlrpc.mk#L9). Should I switch to 1.59.02?

Reply:
I was trying hard to avoid adding a patch :-) But I can do it.